chukwuemekachm/utility_manager_web
# Prof: implement continuous integration using circleci

#### What does this PR achieve?
- Adds CircleCI config to the app 
- adds scripts for testing ts-types and Linting
- fixes failing type checks 

#### Any background context?
- CIrcleCI would help us easily know when  our linting and code is valid and when it is not before the code is merged

#### Manual Testing Steps
- Click on the link to builds below PR and check that they are passing 
- Here is a [link](https://app.circleci.com/github/chukwuemekachm/utility_manager_web/pipelines/c785f432-d97a-41ea-a1d2-d245c9dfd200/workflows/46156c74-6b84-44a6-b674-9c879fc760ae) to a build that was used to test failing linting

#### Relevant Trello Tickets
- [CirceCI config](https://trello.com/c/He1fuAE4/56-d-2-implement-continuous-integration-for-front-end)
